packages/README.md

```diff
@@ -0,0 +1,24 @@
+# planutils packages
+
+Each package must contain the files `manifest.json`, `install`, and `uninstall`. Optionally, `run` can be included if the package installs a command-line executable (e.g., planners). An example of all four files can be found in the `TEMPLATE` directory, and a description of each follows.
+
+## manifest.json
+
+Details on the package. Must include:
+
+1. **name**: Long-form name of the package.
+2. **shortname**: Short-form name that will be used for installation, running, etc.
```

Comment:
Are there any restrictions on the short name as opposed to the long name? For example, no spaces, only ascii, ...?

Reply:
It's now abstracted to just being the folder name.